### ISSUE TITLE
Qview - Throws warning for mosaics containing tracking table

### ISSUE THREAD

**Issue by lwellerastro** [DOI-USGS/ISIS3#3685]
**ISIS version(s) affected**: isis3.10.0

**Description**  
Qview throws warning (and never stops) for a 10+ year old mosaic that used to have a tracking band that was removed sometime in the past, but still contains the table associated with it (InputImages).

See attached error message. 
![QviewError_TrackingTable](https://user-images.githubusercontent.com/34550733/74764394-db5d5b80-523e-11ea-8723-93984b0fdb14.png)

Every time I click OK to make the warning go away, a new one pops up.  Sometimes the only way to get out of qview is to kill it.

**How to reproduce**  
qview /archive/projects/lmmp/rdr/SAB/Mosaic/LRO_NAC_Mosaic_60S200E_50cmp.cub
Click anywhere in the qview window (not even the image) and the warning will pop up.

**Possible Solution**  

**Additional context**  
The program cubeatt (from=mosaic+1) does not work as the tracking band is not present.  I also tried to run the program trackextract to help remove the table but that program does not work on 16 bit files.

There have been some other suggestions made via an astrodiscuss post I put up in regard to this problem ([https://astrodiscuss.usgs.gov/t/how-to-remove-a-table-from-an-isis-cube-and-its-label/372]) that I might try at another time.

**Comment by lwellerastro** [comment]
A work around for this problem is to run cubeatt in the following manner:

> cubeatt from=old.cub to=new.cub proptables=false

Qview should still probably ignore the table though if it exists, or just provide a more meaningful warning and only once, not repeatedly as it is doing now.

**Comment by jessemapel** [comment]
So it looks like qview thinks that the mosaic is the tracking cube itself and is trying to convert the pixels from the mosaic to filenames and fails repeatedly. I think the fix here is to have qview not raise warnings on failure in AdvancedTrackTool::TrackMosaicOrigin and instead just pass. This will cause mosaics like this to just ignore the tracking table. This will also cause qview to silently fail when other issues with the tracking table or cube arise.

Another option would be to have AdvancedTrackTool::TrackMosaicOrigin instead raise a fatal error and die. This would help avoid silent failures, but would be more intrusive to users.

**Comment by lwellerastro** [comment]
Hi, I was just revisiting the data in my original post and it appears to still have the problem I posted.

I'm working in 3.10.2.  It's been so long I can't remember testing this, but I assumed it worked when this post was closed.  Should I expect it work under 3.10.2 or only under a version of isis4?

How to reproduce
qview /archive/projects/lmmp/rdr/SAB/Mosaic/LRO_NAC_Mosaic_60S200E_50cmp.cub
Click anywhere in the qview window (not even the image) and the warning will pop up.
